generators/entity-client/templates/angular/src/main/webapp/app/entities/entity.service.ts.ejs

```diff
@@ -45,91 +45,86 @@ export class <%= entityAngularName %>Service {
     <%_ } _%>
 
     constructor(private http: HttpClient) { }
-    <%_ if (entityAngularName.length <= 30) { _%>
 
+    <%_ if (entityAngularName.length <= 30) { _%>
     create(<%= entityInstance %>: I<%= entityAngularName %>): Observable<EntityResponseType> {
     <%_ } else { _%>
-
     create(<%= entityInstance %>: I<%= entityAngularName %>):
         Observable<EntityResponseType> {
     <%_ } _%>
-        const copy = this.convert(<%= entityInstance %>);
-        return this.http.post<I<%= entityAngularName %>>(this.resourceUrl, copy, { observe: 'response' })
-            .map((res: EntityResponseType) => this.convertResponse(res));
+        <%_ if (hasDate) { _%>
+        const copy = this.convertDateFromClient(<%= entityInstance %>);
+        <%_ } _%>
+            return this.http.post<I<%= entityAngularName %>>(this.resourceUrl,
+                    <%_ if (hasDate) { _%> copy <%_ } else { _%> <%= entityInstance %> <%_ } _%>,
```

Comment:
`<%_ _%>` should be used only for newline templates, for inline templates use `<% %>`, it doesnt matter in ts files due to prettier, but we would like to be consistent as it matters for other file types